.github/workflows/lint.yml

```diff
@@ -88,6 +88,13 @@ jobs:
           level: warning
           flags: --linelength=120 --exclude=java/src/main/native/*.c
           filter: "-runtime/references"
+      - name: Run clang-tidy
```

Comment:
I suggest not using 3rd-party actions when possible, for the possibility of leaking tokens.

Reply:
Good call. Although IIRC the token is periodically rotated? We are using a few 3rd party checks so that would be an all or non situation. Would it make sense for us to verify the token access to make sure "posting to the PR thread" (and other necessary access) are the only things allowed?